packages/aft/test/version_bump/data/snapshots/multi_package_update.diff

```diff
@@ -12,8 +12,6 @@ diff --git a/packages/amplify/amplify_flutter/pubspec.yaml b/packages/amplify/am
 +version: 2.4.0
 -  amplify_core: ">=2.3.0 <2.4.0"
 +  amplify_core: ">=2.4.0 <2.5.0"
--  aws_common: ">=0.7.1 <0.8.0"
-+  aws_common: ">=0.7.2 <0.8.0"
```

Comment:
Note to reviewers: The constraints on aws_common do not need to be updated since this is not a breaking change. The existing version constraint of `">=0.7.1 <0.8.0"` is still valid (`">=0.7.0 <0.8.0"` would have also been valid). A customer can run pub upgrade to get the latest version of aws_common.

I have added a new test case (see .diff below) where aws_common receives a breaking change, bumping the range to `">=0.8.0 <0.9.0"`, which would result in an update to this file.